backend/apis/clients/add-client.api.js

```diff
@@ -262,12 +257,23 @@ app.post("/api/clients", (req, res, next) => {
             }
 
             connection.commit();
+            returnTheClient();
+          });
+        });
+
+        function returnTheClient() {
```

Comment:
The call to create a client is now returning the client object after it succeeds in creating it.